storey/flow.py

```diff
@@ -147,15 +150,23 @@ class Event:
 
     def __init__(self, body, key=None, time=None, id=None, headers=None, method=None, path='/', content_type=None, awaitable_result=None):
         self.body = body
-        self.id = id or uuid.uuid4().hex
         self.key = key
         self.time = time or datetime.now(timezone.utc)
+        self.id = id or uuid.uuid4().hex
         self.headers = headers
         self.method = method
         self.path = path
         self.content_type = content_type
         self._awaitable_result = awaitable_result
 
+    def __eq__(self, other):
+        if not isinstance(other, Event):
+            return False
+
+        result = self.body == other.body and self.time == other.time and self.id == other.id and self.headers == other.headers
+        result = result and self.method == other.method and self.path == other.path and self.content_type == other.content_type
```

Comment:
why break it to two statements just break the line

Reply:
Actually pycharm and the linter disagree on the indentation level in that case, so this is my workaround...